app/javascript/lib/vue-components/modules/BeesWarmChart.vue

```diff
@@ -100,13 +105,25 @@ export default {
         this.deepCloneData(newValue);
         this.updateData = true;
       }
+    },
+    $route(to, from) {
+      if (to !== from) {
```

Comment:
This comparison is worthless. Objects cannot be compared like primitives

Reply:
Done!